### PR TITLE
Disable inbound connections on mobiles

### PR DESCRIPTION
**Pull request by dshulyak**
Since we are not using p2p server v4 discovery and dialing i had to add another control option to p2p config

Closes: https://github.com/status-im/status-go/issues/978

### CODE REVIEW

File: geth/node/node_test.go
Comment (adambabik) [inline geth/node/node_test.go]: Not sure if this `assert` makes sense. If the p2p server fails to start, it's a problem with configuration or system resources. What's the advantage of continuing with the rest of the test?

File: geth/node/node_test.go
Comment (adambabik) [inline geth/node/node_test.go]: Maybe we can try `t.Parallel()`?

**Review by adambabik — APPROVED** [review]
LGTM provided that CI passes.

**Comment by adambabik** [comment]
What's the status of this PR?

**Comment by dshulyak** [comment]
need to push a couple of fixed suggestions and test it on mobile before merging

**Comment by dshulyak** [comment]
I will restore it later

**Pull request closed without merging**
